k8s-operator/apis/v1alpha1/types_connector.go

```diff
@@ -98,12 +147,18 @@ type ConnectorStatus struct {
 	// +listMapKey=type
 	// +optional
 	Conditions []ConnectorCondition `json:"conditions"`
-	// SubnetRouter status is the current status of a subnet router
+	// SubnetRouter status is the current status of a subnet router if defined on this Connector.
 	// +optional
 	SubnetRouter *SubnetRouterStatus `json:"subnetRouter"`
+	// ExitNode status is the current status of an exit node if defined on this Connector.
+	// +optional
+	ExitNode *ExitNodeStatus `json:"exitNode"`
+	// IsExitNode is set to true if a Connector defines an exit node.
+	// +optional
+	IsExitNode bool `json:"isExitNode"`
```

Comment:
This field exists so that we can have a value for [the additional printer column](https://github.com/tailscale/tailscale/pull/10724/commits/b5b5ae25a081274a6fe682e769fd946991ea9f37#diff-b5e4ef42a37ef6da3980eb4c5c233e209597bbe4681d9aebebc54822301f9e06R21) that shows if a Connector is an exit node on `kubectl get`

```
$ kubectl get connector
NAME         SUBNETROUTES   ISEXITNODE   STATUS
exitnode                    true         ExitNodeCreated
exposepods   10.40.0.0/14   false        SubnetRouterCreated
```